### PR TITLE
fix for 2.10

### PR DESCRIPTION
**Pull request by dommrogers**

### CODE REVIEW

**Comment by dommrogers** [comment]
#rem .zip

**Comment by FINDarkside** [comment]
Thanks for the PR, can @zeobviouslyfakeacc or @ds5678 check this?

**Comment by ds5678** [comment]
I was waiting until I pushed a new NuGet package for the game.

File: DeveloperConsole/InitializePatch.cs [hunk 1]
```diff
@@ -1,20 +1,17 @@
...
 using UnityEngine;
-using UnityEngine.ResourceManagement.AsyncOperations;
+using UnityEngine.AddressableAssets;
 
-namespace DeveloperConsole {
+namespace DeveloperConsole
+{
 
-    [HarmonyPatch(typeof(BootUpdate), "Start")]
+	[HarmonyPatch(typeof(BootUpdate), "Start")]
```
Comment (ds5678) [inline DeveloperConsole/InitializePatch.cs]: Indentation

File: DeveloperConsole/InitializePatch.cs [hunk 2]
```diff
@@ -1,20 +1,17 @@
 ﻿using HarmonyLib;
 using Il2Cpp;
-using Il2CppTLD.AddressableAssets;
 using UnityEngine;
-using UnityEngine.ResourceManagement.AsyncOperations;
+using UnityEngine.AddressableAssets;
 
-namespace DeveloperConsole {
+namespace DeveloperConsole
+{
```
Comment (ds5678) [inline DeveloperConsole/InitializePatch.cs]: I'm not a fan of Java style bracketing either, but you should always adhere to the existing style when contributing.

**Review by ds5678 — CHANGES_REQUESTED** [review]
@dommrogers Simple review. In addition to the two small formatting changes, I also want you to bump the nuget reference version.

File: DeveloperConsole/InitializePatch.cs (hunk 1 above)
Comment (dommrogers) [reply DeveloperConsole/InitializePatch.cs]: my VS was auto formatting.. turned it off.

File: DeveloperConsole/InitializePatch.cs (hunk 2 above)
Comment (dommrogers) [reply DeveloperConsole/InitializePatch.cs]: my VS was auto formatting.. turned it off.

**Review by ds5678 — APPROVED** [review]

**Comment by dommrogers** [comment]
[DeveloperConsole.zip](https://github.com/FINDarkside/TLD-Developer-Console/files/11127912/DeveloperConsole.zip)

new file

**Pull request merged**